docs/ref/request-response.txt

```diff
@@ -1070,18 +1070,33 @@ types of HTTP responses. Like ``HttpResponse``, these subclasses live in
     (e.g. ``'https://www.yahoo.com/search/'``), an absolute path with no domain
     (e.g. ``'/search/'``), or even a relative path (e.g. ``'search/'``). In that
     last case, the client browser will reconstruct the full URL itself
-    according to the current path. See :class:`HttpResponse` for other optional
-    constructor arguments. Note that this returns an HTTP status code 302.
+    according to the current path.
+
+    The constructor accepts an optional keyword argument
+    ``preserve_request_method``, which defaults to ``False``. See
+    :class:`HttpResponse` for other optional constructor arguments.
+
+    This uses a HTTP status code 302 when ``preserve_request_method=False``, and
+    a HTTP status code 307 when ``preserve_request_method=True``.
```

Comment:
Been re-reading this and I think we can improve these docs to be clearer including what `preserve_request_method` means

I also want a few more folks to review, so I will put the ticket back in the review queue :+1: